src/api/vdom.js

```diff
@@ -140,17 +140,19 @@ const attributesContext = propContext(attributes, {
 });
 
 function resolveTagName (tname) {
-  // If the tag name is a function, a Skate constructor or a standard function
-  // is supported.
-  //
-  // - If a Skate constructor, the tag name is extracted from that.
-  // - If a standard function, it is used as a helper.
-  if (typeof tname === 'function') {
-    return tname[$name] || tname;
+  if (!tname) {
+    return tname;
```

Comment:
Is there some benefit of doing this instead of just returning `undefined` or `null`? Seems a bit weird.